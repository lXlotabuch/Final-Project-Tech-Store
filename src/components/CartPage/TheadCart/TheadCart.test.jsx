import {render} from '@testing-library/react'
import shallow from 'enzyme/build/shallow'
import React from 'react'
import { machMedia } from '../../../mocks/matchMedia.mock'
import TheadCart from './TheadCart'

describe('TheadCart', () => {
  beforeEach(() => {
    machMedia()
  })
  test('render', () => {
    const {asFragment} = render(<TheadCart />)
    expect(asFragment()).toMatchSnapshot()
  })
  test('to have .subtotal', () => {
    const component = shallow(<TheadCart />)
    const wrapper = component.find('.subtotal')
    expect(wrapper.length).toBe(1)
  })
})