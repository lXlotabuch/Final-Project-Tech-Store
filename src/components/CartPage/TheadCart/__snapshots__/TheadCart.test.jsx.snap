// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheadCart render 1`] = `
<DocumentFragment>
  <div
    class="sc-bdfBwQ dVziVR"
  >
    <div
      class="ant-row"
    >
      <div
        class="ant-col ant-col-5"
      >
        Product(s)
      </div>
      <div
        class="ant-col ant-col-7"
      />
      <div
        class="ant-col ant-col-3 price"
      >
        Price
      </div>
      <div
        class="ant-col ant-col-5 qty"
      >
        Quantity
      </div>
      <div
        class="ant-col ant-col-3 subtotal"
      >
        Subtotal
      </div>
      <div
        class="ant-col ant-col-1"
      />
    </div>
  </div>
</DocumentFragment>
`;
